Neos.Flow/Tests/Unit/ObjectManagement/Proxy/CompilerTest.php

```diff
@@ -69,27 +69,27 @@ public function annotationsAndStrings()
                 '@\Neos\Flow\Annotations\Session'
             ],
             [
-                new Validate(['value' => 'foo1', 'type' => 'bar1']),
+                new Validate('foo1', 'bar1'),
                 '@\Neos\Flow\Annotations\Validate(type="bar1", argumentName="foo1")'
             ],
             [
-                new Validate(['type' => 'bar1', 'options' => ['minimum' => 2]]),
+                new Validate(null, 'bar1', ['minimum' => 2]),
```

Comment:
This shows that the choice to have `argumentName` as the anonymous argument was probably suboptimal. I would rather consider the validate type to be anonmyous - maybe something to consider for next major?

Reply:
No idea on where validation is used more. But I have no issue with `@Flow\Validate("$myArg", type="NotEmpty")`, TBH. What is _not_ straighforward with that?